setup.py

```diff
@@ -64,6 +64,7 @@
 import setup_ts
 
 release = os.environ.get('NNI_RELEASE')
+jupyter_lab_version = os.popen('jupyter lab --version').read().rstrip('\n').split('.')[0]
```

Comment:
Packages are commonly built on one system and installed on another. So we cannot auto-detect environment in build script. Use command args or environment variable instead.

Reply:
Suggest putting this into a function called `get_jupyter_lab_version`. Because we are making jupyter an optional dependency for building and users may not have jupyter installed.

Reply:
changed